Library/Formula/fuseki.rb

```diff
@@ -73,14 +61,21 @@ def plist; <<-EOS.undent
         <true/>
         <key>KeepAlive</key>
         <false/>
+        <key>EnvironmentVariables</key>
+        <dict>
+           <key>FUSEKI_BASE</key>
+           <string>#{var}/fuseki</string>
+        </dict>
         <key>ProgramArguments</key>
         <array>
-          <string>#{opt_bin}/fuseki-server</string>
-          <string>--config</string>
-          <string>/usr/local/etc/fuseki.ttl</string>
+          <string>#{bin}/fuseki-server</string>
```

Comment:
This should remain `#{opt_bin}` so the `plist` doesn't have to be replaced by the user every update.